daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/PackageLookup.scala

```diff
@@ -17,6 +17,9 @@ object PackageLookup {
       dataTyp <- defn match {
         case _: DValue =>
```

Comment:
Could you put the error cases at the end of the pattern matching ?

Reply:
ok (but this was actually the way the code was before my changes!)
done